FibonacciNumbersWithGenerators.py

```diff
@@ -0,0 +1,18 @@
+def fibonacciGenerator(n = None):
```

Comment:
Python uses snake_case for function names.
```suggestion
def fibonacci_generator(n = None):
```

Reply:
Thanks for the sugestions